### DISCUSSION TITLE
Scripts not working with Fig

### DISCUSSION THREAD

**Discussion by raguay** [johnlindquist/kit#1167] [Q&A]
If you have https://fig.io installed on your system, many of the scripts will not work since a full login shell is ran. Fig will try to open a subshell that interferes with the running of the kit script. Does Script Kit set a environment variable that says it's running the script? If so, I can wrap the calls for fig in an if statement.

**Comment by johnlindquist** [comment]
@raguay I assume you mean the scripts won't run in a terminal running Fig and that the scripts run in the app still work?

You can the the source of the "wrappers" in ~/.kenv/bin. It should set a `KENV`, but it doesn't export it for the session. Can you explain a bit more what your expectation is with code and what you're doing?

**Reply by raguay** [reply]
No, I mean the scripts with a $`` to run a command line aren't working due to the command invoking fig in my .zshrc. If Script Kit would set an environment variable that is unique to it, I can then not run those commands when encountered. Then, all the Script Kit scripts should work okay.

Doesn't anyone else use fig?
